test/rpc_test.py

```diff
@@ -16,13 +16,13 @@
 from torch.distributed.rpc.internal import PythonUDF, _internal_rpc_pickler
 
 
-def requires_process_group_agent(func):
-    from torch.distributed.rpc.api import _agent
-
-    return unittest.skipUnless(
-        isinstance(_agent, ProcessGroupAgent),
-        "Only ProcessGroupAgent supports global termination detection",
-    )
+def requires_process_group_agent(message=""):
+    def decorator(old_func):
+        return unittest.skipUnless(
+            TEST_CONFIG.backend == RpcBackend.PROCESS_GROUP,
```

Comment:
This is still `TEST_CONFIG.rpc_backend` on master, where did we change that to `TEST_CONFIG.backend`?

Reply:
Rebasing